ui/src/androidTest/java/com/hyperwallet/android/transfermethod/ui/PayPalTest.java

```diff
@@ -128,8 +128,8 @@ public void testAddTransferMethod_displaysFeeElementsOnTmcResponse() {
                 matches(withText(R.string.add_transfer_method_fee_label)));
         onView(withId(R.id.add_transfer_method_processing_label)).check(
                 matches(withText(R.string.add_transfer_method_processing_time_label)));
-        onView(withId(R.id.add_transfer_method_fee_value)).check(matches(withText("USD 0.25")));
```

Comment:
Add this back as well. Currently it's not returning on qamaster but we want to try to provide mocked reponses that will be close to the ones used.

Reply:
@skoong Could you assist how to provide it via mock, please?